rkyv/src/util/aligned_vec.rs

```diff
@@ -348,6 +348,38 @@ impl AlignedVec {
             }
         }
     }
+    /// Resizes the Vec in-place so that len is equal to new_len.
+    ///
+    /// If new_len is greater than len, the Vec is extended by the difference, with each additional slot filled with value. If new_len is less than len, the Vec is simply truncated.
+    /// 
+    /// # Panics
+    ///
+    /// Panics if the new length exceeds `usize::MAX` bytes.
+    ///
+    /// # Examples
+    /// ```
+    /// use rkyv::AlignedVec;
+    ///
+    /// let mut vec = AlignedVec::new();
+    /// vec.push(3);
+    /// vec.resize(3, 2);
+    /// assert_eq!(vec.as_slice(), &[3, 2, 2]);
+    ///
+    /// let mut vec = AlignedVec::new();
+    /// vec.extend_from_slice(&[1, 2, 3, 4]);
+    /// vec.resize(2, 0);
+    /// assert_eq!(vec.as_slice(), &[1, 2]);
+    /// ```
+    #[inline]
+    pub fn resize(&mut self, new_len: usize, value: u8) {
+        let len = self.len;
```

Comment:
Why save out `len` here? We should be able to just use `self.len` throughout since `reserve` doesn't modify it.